play-ahc-ws-standalone/src/main/java/play/libs/ws/ahc/StandaloneAhcWSRequest.java

```diff
@@ -339,6 +357,16 @@ String getVirtualHost() {
         return executor.apply(this);
     }
 
+    @Override
+    public CompletionStage<? extends StandaloneWSResponse> stream() {
+        WSRequestExecutor executor = foldRight(r -> {
+            StandaloneAhcWSRequest ahcWsRequest = (StandaloneAhcWSRequest) r;
+            Request ahcRequest = ahcWsRequest.buildRequest();
+            return client.executeStream(buildRequest(), materializer.executionContext());
```

Comment:
enable request filters for requests that go through stream rather than execute()